include/SW_Carbon.h

```diff
@@ -23,7 +23,7 @@ void SW_CBN_deconstruct(void);
 void SW_CBN_read(SW_CARBON* SW_Carbon, SW_MODEL* SW_Model, char *InFiles[],
                  LOG_INFO* LogInfo);
 void SW_CBN_init_run(VegType VegProd_veg[], SW_MODEL* SW_Model,

```

Comment:
Now that SW_MODEL regained time info, we can go back to `SW_CBN_init_run(VegType VegProd_veg[], SW_MODEL* SW_Model, SW_CARBON* SW_Carbon, LOG_INFO* LogInfo)`